src/utils/S3/deleteFromS3.ts

```diff
@@ -0,0 +1,30 @@
+import { S3 } from "aws-sdk";
+import { accessKeyId, secretAccessKey } from "./config";
+import { S3DeleteResponse } from "./types";
+
+const s3: S3 = new S3({
+  accessKeyId,
+  secretAccessKey,
+});
+
+const deleteFromS3 = async ({
+  file_name,
+  bucket_name,
+}): Promise<S3DeleteResponse> => {
+  const params = {
+    Bucket: bucket_name,
+    Key: file_name,
+  };
+
+  const deleteImage = s3.deleteObject(params).promise();
```

Comment:
Do we have to do this like this?

Seems like there should be a better approach than using the `promise()` call.

I would say, consider using the callback.

```javascript
return s3.deleteObject(params, (err, data) => {
  if(err){...}
  else {...}
});
```

Reply:
callback works as well actually, I used callback at first but then switched to `.promise()`. will be updated soon